### PR TITLE
Add Logic to load profile info to firebase

### PR DESCRIPTION
**Pull request by NCMoseley**

### CODE REVIEW

File: js/screens/EditProfile/EditProfile.js [hunk 1]
```diff
@@ -6,98 +6,197 @@ import {
...
+        <TouchableHighlight
+          underlayColor={colors.borealisGreen}
+          activeOpacity={1}
+          onPress={() => {
+            updateWS("collaborative");
+          }}
+        >
+          <View style={styles.workstyleIcons}>
+            <Image
+              style={{ height: 80, width: 80 }}
```
Comment (mandiwise) [inline js/screens/EditProfile/EditProfile.js]: No inline styles.

File: js/components/Chips/Chips.js [hunk 2]
```diff
@@ -10,44 +10,44 @@ const Chips = ({
...
+    {
+      <TextInput
+        type="text"
+        autoCorrect={false}
+        value={value}
+        ref={inputRef}
+        autoCapitalize={"words"}
+        style={styles.chipsInput}
+        placeholder={placeholder}
+        onChangeText={text => {
```
Comment (mandiwise) [inline js/components/Chips/Chips.js]: `onChange={setText}` is the same thing with less code.

File: js/screens/EditProfile/EditProfileContainer.js [hunk 3]
```diff
@@ -1,32 +1,169 @@
...
+  static propTypes = {
+    navigation: PropTypes.object.isRequired
+  };
+  static navigationOptions = ({ navigation }) => ({
+    title: "EDIT PROFILE",
+    ...headerBarStyle(navigation, true)
+  });
   constructor(props) {
     super(props);
+    const {
```
Comment (mandiwise) [inline js/screens/EditProfile/EditProfileContainer.js]: Setting state from props is anti-pattern.

File: js/screens/EditProfile/EditProfileContainer.js [hunk 4]
```diff
@@ -1,32 +1,169 @@
...
+  setChips = chips => {
+    if (chips && chips.length) {
+      this.setState({ chips });
+    }
+  };
+
+  submit = async () => {
+    this.props.dispatch(
+      updateProfileOnEdit({
+        fullname: this.state.fullname,
```
Comment (mandiwise) [inline js/screens/EditProfile/EditProfileContainer.js]: Using internal state to update your Redux store is not the right thing. If you're using Redux, then use Redux to track this state!

**Review by mandiwise — CHANGES_REQUESTED** [review]
Major issues in the EditProfileContainer.js file.

**Comment by NCMoseley** [comment]
please review

**Pull request merged**